examples/few-shot/maml_omniglot.py

```diff
@@ -50,11 +50,10 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
+from support.omniglot_loaders import OmniglotNShot
```

Comment:
- Please update all the examples (e.g. `from .helpers.xxx import yyy`).
- The local package import should be placed after the `import torchopt`.

Reply:
The import order is autofixed by isort I think.

Reply:
> The import order is autofixed by isort I think.

Add comment `# isort: skip`.